### PR TITLE
Homework14: Ekaterina Zavadskaya

### PR DESCRIPTION
**Pull request by KateZavadskaya**
Add hm14

hm14_1.py - done
hm14_2.py - done as comment and via eval 

Docs: hm14_1.py, hm14_2.py

### CODE REVIEW

**Comment by KateZavadskaya** [comment]
@Stml89 local Pylint is ok 💆‍♀️
Please, take a look.

hm14_2.py master Yoda, I'm exhausted( Done via eval😞 and as comment becouse of Linters..

**Comment by Stml89** [comment]
Why are files from HW11 present in HW14?

File: hm14_2.py
Comment (Stml89) [inline hm14_2.py:1]: how should I rate this task?

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hm14_2.py
Comment (KateZavadskaya) [reply hm14_2.py:1]: HM has 2 tasks. 1 done, 2 done not according to the rules. So, it seems as 50% from hm 🫠.
Files from HW11 really exist in HM14🙈 Can you, please, make advice, have it is better to delete? Should I close this pull request and create new one?

Comment (KateZavadskaya) [reply hm14_2.py:1]: Delete local files, create new commit, push.
Pylint have already discussed, Ruff - my side is ok.
Please, check and correct my actions if I am wrong.

Comment (Stml89) [reply hm14_2.py:1]: Using `ruff check .` command instead of `ruff --output-format=github .` to fix the Ruff issue

Comment (Stml89) [reply hm14_2.py:1]: > HM has 2 tasks. 1 done, 2 done not according to the rules. So, it seems as 50% from hm 🫠. Files from HW11 really exist in HM14🙈 Can you, please, make advice, have it is better to delete? Should I close this pull request and create new one?

In my world, there are only 2 possible cases: 
1\ task is done
2\ task is not done


Based on this, I can say that 2nd task was not done according to the requirements

File: hm14_1.py
Comment (Stml89) [inline hm14_1.py:1]: Let's give the module (file) better names next time.

**Comment by Stml89** [comment]
Pylint hightailed 2 issues:
```
hm14_1.py:44:48: E0606: Possibly using variable 'avagare_1' before assignment (possibly-used-before-assignment)
hm14_1.py:45:49: E0601: Using variable 'avagare_2' before assignment (used-before-assignment)
```

**Review by Stml89 — APPROVED** [review]
Approved 1 task out of 2

**Pull request merged**